apis/logging/v1/cluster_log_forwarder.go

```diff
@@ -58,16 +62,18 @@ func (status ClusterLogForwarderStatus) IsReady() bool {
 	return true
 }
 
-// IsDegraded returns true if any of the subordinate conditions are degraded.
-func (status ClusterLogForwarderStatus) IsDegraded() bool {
+// IsAllNotReady checks to see if all subordinate conditions are not ready
+func (status ClusterLogForwarderStatus) IsAllNotReady() bool {
```

Comment:
Consider flipping the naming and meaning as it its possible to write an "if" as a double negative which is sometime challenging to wrap your head around:
```
if !IsAllNotReady {
}
```
vs
```
if !IsAllReady {
}
```

Reply:
This function is not needed anymore and was removed.